docs/source/release_notes/release-notes.rst

```diff
@@ -42,8 +42,9 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |improved| |deprecated|
+         - `AssertUtils` logging methods will now ask for a sfl4j logger to log to, instead of using a default logger.
+           This should make log events from AssertUtils easier to filter.
```

Comment:
Dumping all errors into a single logger makes it much harder to triage errors to the correct team responsible.  We use these common AssertUtils methods in a huge number of places internally and as it stands now it requires someone to manually look at the error to be able to triage it to the correct internal team.  It also means that we can't just edit our logging config to ignore some specific errors or split them into different files (which all becomes possible when you use a separate logger for different errors).